Jenkinsfile-nightly

```diff
@@ -1,8 +1,9 @@
-OLTP_TERMINALS = ['1','2','4','8','16']
-
 pipeline {
     agent none
 
+    environment{
+        PSS_CREATOR= credentials('pss-creator')
```

Comment:
Add a comment to explain what this is and why people should not change it.

Reply:
Done